<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastreBrowser</class>
 <widget class="QWidget" name="CadastreBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qadastre browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="PDFGraphicView" name="graphicsView"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="status">
     <property name="text">
      <string>Hello world !</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PDFGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>pdfgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
